lms/djangoapps/teams/tests/test_views.py

```diff
@@ -186,7 +202,7 @@ def make_call(self, url, expected_status=200, method='get', data=None, content_t
 
         If a user is specified in kwargs, that user is first logged in.
         """
-        user = kwargs.pop('user', 'student_enrolled')
+        user = kwargs.pop('user', 'student_enrolled_not_on_team')
```

Comment:
@cahrens sorry looks like our replies crossed. I created https://openedx.atlassian.net/browse/TNL-2960 to address that case. Let me know your feedback on the approach.